### ISSUE TITLE
Operations on a 2d tensor zeroes many of its elements

### ISSUE THREAD

**Issue by mnts-i** [tensorflow/tfjs#4574]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow.js): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: Unknown
- TensorFlow.js installed from (npm or script link): script link (CDN)
- TensorFlow.js version (use command below): 2.8.4
- Browser version: Chrome
- Tensorflow.js Converter Version: -

**Describe the current behavior**
I was writing a simple normalise function in order to normalise a 2d tensor which contains approx ~21k integers which were fed from an array. After some training and testing, I noticed that the loss and the predictions were way off, so I started logging stuff in my code. What I found was after a simple operation on my tensor (even a simple subtraction) the operation would work correctly until the 16.383th element after which all the other elements would become zero.

**Describe the expected behavior**
Perform the tensor operations without zeroing the elements.

**Standalone code to reproduce the issue**
https://jsfiddle.net/j2wu8f59/

Open the console and run the jsfiddle. There you'll see 3 logs, the original array (25K length), the 2d tensor created using the array's values and finally the normalised tensor. If you check the original array and tensor you'll see that all the values are set (unique) but if you inspect the normalised tensor you'll see that the 16.384th element and beyond all are zeros.

**Comment by lina128** [comment]
Hi @pyu10055 I ran the jsfiddle, the normalized tensor has all the values, so I'm guessing it's a window specific issue. Can you check on a Windows machine?

**Comment by pyu10055** [comment]
I tried your example on a windows 10 machine, the output tensor is complete. This might be related to the texture limit on your browser. @mnts-i can try [this page](https://js.tensorflow.org/debug/) and the output with us?
